library-api/src/test/groovy/io/opentelemetry/instrumentation/api/tracer/HttpClientTracerTest.groovy

```diff
@@ -195,6 +197,16 @@ class HttpClientDecoratorTest extends ClientDecoratorTest {
       protected String responseHeader(Map m, String name) {
         return m[name]
       }
+
+      @Override
+      protected HttpTextFormat.Setter<Map> getSetter() {
+        return null
```

Comment:
Do we need to add any test to exercise this functionality?

Reply:
Per offline discussion with @trask, this is just cleaning up.  Adding tests will be a different task.

Reply:
@heyams is this tracked in #912?

Reply:
Not yet.

Reply:
Ok, created #986